explorer/config-loader.js

```diff
@@ -0,0 +1,25 @@
+var fs = require('fs');
+
+getConfig = function () {
```

Comment:
Can't you just export the config object instead of the function to load the config?

Reply:
not sure how to do that, export default doesn't work on this version

Reply:
something like:
```
var fs = require('fs');

var config = {};
try {
    var network = process.env.NETWORK || 'testnet';
    var configFilename = 'config.' + network + '.json';
    var configContents = fs.readFileSync(configFilename);
    config = JSON.parse(configContents);
    console.log(configFilename + ' found.');
}
catch (error) {
    if (error.code === 'ENOENT') {
        console.log('No config file found.');
    }
    else {
        throw error;
        process.exit(1);
    }
}

exports = config;
```
 then you just need `var config = require("./config-loader")`